gsa/src/web/pages/tlscertificates/details.js

```diff
@@ -0,0 +1,110 @@
+/* Copyright (C) 2019 Greenbone Networks GmbH
+ *
+ * SPDX-License-Identifier: GPL-2.0-or-later
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+import React from 'react';
+
+import {connect} from 'react-redux';
+
+import _ from 'gmp/locale';
+
+import {isDefined} from 'gmp/utils/identity';
+
+import Comment from 'web/components/comment/comment';
+
+import DateTime from 'web/components/date/datetime';
+
+import DetailsLink from 'web/components/link/detailslink';
+
+import InfoTable from 'web/components/table/infotable';
+import TableData from 'web/components/table/data';
+import TableBody from 'web/components/table/body';
+import TableRow from 'web/components/table/row';
+
+import DetailsBlock from 'web/entity/block';
+import EntityLink from 'web/entity/link';
+import {Col} from 'web/entity/page';
+
+import {
+  loadEntity as loadTlsCertificate,
+  selector as tlsCertificateSelector,
+} from 'web/store/entities/tlscertificates';
+
+import compose from 'web/utils/compose';
+import withGmp from 'web/utils/withGmp';
+import PropTypes from 'web/utils/proptypes';
+import {renderYesNo} from 'web/utils/render';
+
+const TlsCertificateDetails = ({entity, links = true}) => {
+  console.log('enityt details', entity);
```

Comment:
```suggestion
```

Reply:
This should have been gone already, it's not in my local changes anymore and I rebased everything. This is weird...